client/src/screens/home/Index.js

```diff
@@ -0,0 +1,155 @@
+
+import CURRENT_USER_QUERY from '../../graphql/current-user.query';
+
+import { Center, Container, Holder } from '../../components/Container';
+import { ListItem } from '../../components/List';
+import { Progress } from '../../components/Progress';
+import { Segment } from '../../components/Segment';
+
+class Home extends Component {
+  static navigationOptions = {
+    title: 'Home',
+    tabBarIcon: ({ tintColor }) => <Icon size={34} name="home" color={tintColor} />,
+  };
+
+  state = {
+    selectedIndex: 0,
+  };
+
+  handleNavigate = () => {
+    this.props.navigation.navigate('Edit');
+  };
+
+  handleIndexChange = (index) => {
+    this.setState({
+      ...this.state,
+      selectedIndex: index,
+    });
+  };
+
+  handleEventPress = (event) => {
+    this.props.navigation.navigate('Event', { id: event.item.id, title: event.item.name });
+  };
+
+  handleSchedulePress = () => {
+    this.props.navigation.navigate('Availability');
+  };
+
+  render() {
+    const { loading, user } = this.props;
+
+    if (loading || !user) {
+      return (
+        <Container>
+          <Progress />
+        </Container>
+      );
+    }
+
+    return (
+      <Container>
+        <Holder margin marginBot transparent>
+          <Segment
+            values={['Events', 'Requests']}
+            handleIndexChange={this.handleIndexChange}
+            selectedIndex={this.state.selectedIndex}
+          />
+        </Holder>
+        {this.state.selectedIndex === 0 ? (
+          <View style={{ flexDirection: 'row' }}>
+            <FlatList
+              data={user.events}
+              ListHeaderComponent={() =>
+                (!user.events.length ? (
+                  <Center>
+                    <Text>There are no events currently.</Text>
+                  </Center>
+                ) : null)}
+              keyExtractor={event => event.id}
+              renderItem={event => (
+                <ListItem
+                  title={event.item.name}
+                  icon="bullhorn"
+                  onPress={() => this.handleEventPress(event)}
+                />
+              )}
+              refreshing={this.props.networkStatus === 4}
+            />
+          </View>
+        ) : (
+          <View style={{ flexDirection: 'row' }}>
+            <FlatList
+              data={user.schedules}
+              ListHeaderComponent={() =>
+                (!user.schedules.length ? (
+                  <Center>
+                    <Text>There are no requests available. Make sure you have joined a group.</Text>
+                  </Center>
+                ) : null)}
+              keyExtractor={schedule => schedule.id}
+              renderItem={schedule => (
+                <ListItem
+                  title={schedule.item.name}
+                  subtitle={
+                    schedule.item.startTime === 0
+                      ? 'Ongoing'
+                      : `${moment(schedule.item.startTime).format('YYYY-MM-DD')} to ${moment(
```

Comment:
this doesn't seem to be working, everything is coming up 1970

Reply:
Hmm I thought it was just the test data you loaded - I'll fix now.

Reply:
Should be fixed

Reply:
{
          "name": "Wagga Wagga OOA",
          "startTime": 1517837709,
          "endTime": 1518010509
        },

should be, errrm, 

    startTime: MONDAY + (60 * 60 * 24 * 1),
    endTime: MONDAY + (60 * 60 * 24 * 3),

Reply:
That fixed it.